### DISCUSSION TITLE
Accessing front camera

### DISCUSSION THREAD

**Discussion by holoholic** [google/model-viewer#2076] [Ideas]
Is it possible to select the front camera in AR mode in webXR [Chrome for Android and Safari for iOS]?

**Comment by elalish** [comment]
No, that's not yet a feature of WebXR. And while that in itself would be fairly simple, I think what you really want is a face tracking API? That'll take a little while.

**Comment by holoholic** [comment]
@elalish Thanks for your reply. Yes, one of my purpose to use front camera is actually for face tracking and related AR application directly through the browser. I am looking forward for such API to be available in model-viewer in future.

**Comment by Alecsandroo** [comment]
any news on this feature since the last post ?
